### ISSUE TITLE
Ablation study exmperiment

### ISSUE THREAD

**Issue by GanAndreas** [edgarschnfld/CADA-VAE-PyTorch#19]
Hi, first of all, thank you for your work. It's so great!!!

I want to recreate your ablation study by train the model with only CA-VAE and DA-VAE. I tried to do it by disable the distance loss or cross-reconstruction loss in the total loss, but I can't get the result on your paper. I only get harmonic mean accuracy of 2.18% for CA-VAE and 39.52% for DA-VAE. Can you point out my mistake?

Thank you

**Comment by edgarschnfld** [comment]
Hi, it's very much possible that the hyperparameters have to be adjusted for each ablation, especially the early stopping step for the final classifier. That should be done on the validation split (=the split that is not used for the final testing), see here https://github.com/edgarschnfld/CADA-VAE-PyTorch/issues/10.

**Comment by angela000** [comment]
Hi,have you got the true ablation result ? can you show me the details ?
